apps/main/src/dex/components/PageIntegrations/index.tsx

```diff
@@ -71,7 +71,7 @@ const IntegrationsComp = ({
 
       push(pathname)
     },
-    [integrationsTags, push, params, rChainId, searchParams, visibleNetworksList],
```

Comment:
Why was this removed? Now my linter complains it's missing

Reply:
yeah my mistake, I had a refactor that was undone and this stayed wrong